addons/mail/static/src/components/channel_member_list/channel_member_list.js

```diff
@@ -9,14 +9,18 @@ export class ChannelMemberList extends Component {
     /**
      * @returns {Thread}
      */
+    get ChannelMemberListView() {
+        return this.messaging.models['ChannelMemberListView'].get(this.props.localId);
+    }
+
     get channel() {
-        return this.messaging.models['Thread'].get(this.props.channelLocalId);
+        return this.ChannelMemberListView.threadView.thread;
     }
```

Comment:
I don't think we want any getter other than the one that retrieves the record according to its localId

Reply:
this is to avoid useless diff inside the template. I don't know if this rule is strict or not.

Reply:
I believe that the main purpose of introducing view models is to empty the components, this is what Alex does in his recent PRs. I don't know the background of your changes but I feel like it's the same kind of task?